api/routes.go

```diff
@@ -36,6 +36,7 @@ func SetupRoutes(router *gin.Engine) {
 	rest.POST("/collections/:id/activate", CollectionActivateHandler)
 	rest.GET("/content_units/", ContentUnitsListHandler)
 	rest.POST("/content_units/", ContentUnitsListHandler)
+	rest.POST("/content_unit_autoname/", ContentUnitAutoname)
```

Comment:
`/content_units/autoname` is the "correct" REST route

Reply:
move to operations